source/docs/03-metadata.xml

```diff
@@ -525,12 +543,7 @@
                            <specDesc key="biblStruct"/>
                         </specList>
                      </p>
-                     <div xml:id="headerAssociatingMetadataAndData" type="div5">
-                        <head>Associating Metadata and Data</head>
-                        <p>In the MEI header, the <att>data</att> attribute may be used to associate metadata with related notational elements.</p>
-                        <p>Similarly, in the body of the MEI document, the <att>decls</att> attribute may be used to associate parts of the encoded text with related metadata.</p>
-                        <p>The most useful associations of this type are between the bibliographic description of a source and the material taken from it.</p>
-                     </div>
+                     <p/>
```

Comment:
```suggestion
                     <div xml:id="headerAssociatingMetadataAndData" type="div5">
                     <head>Associating Metadata and Data</head>
                        <p>In the MEI header, the <att>data</att> attribute may be used to associate metadata with related notational elements.</p>
                        <p>Similarly, in the body of the MEI document, the <att>decls</att> attribute may be used to associate parts of the encoded text with related metadata.</p>
                        <p>The most useful associations of this type are between the bibliographic description of a source and the material taken from it.</p>
                     </div>
```

Reply:
@bwbohl this will fix it, doesn't it?

Reply:
Okay, then the comment by @bwbohl is resolved and my suggestion can be discarded.